data/test/standalone-topography/rainbow.yaml

```diff
@@ -0,0 +1,36 @@
+structures:
+  - name: stripe
+    structure:
+      palette:
+        'r': "#ff0000"
+        'o': "#ff8800"
+        'y': "#ffff00"
+        'g': "#00ff00"
+        'b': "#00ffff"
+        'i': "#0000ff"
+        'v': "#ff00ff"
+      map: |
+        roygbiv
+mask: '.'
+palette:
+  'r':
+    structure:
+      name: stripe
+    cell: "#ffffff"
+map: |
+  r...............
+  .r..............
+  ..r.............
+  ...r............
+  ....r...........
+  .....r..........
+  ......r.........
+  .......r........
+  ........r.......
+  .........r......
+  ..........r.....
+  ...........r....
+  ............r...
+  .............r..
+  ..............r.
+  ...............r
```

Comment:
I tried what a curve will do:
```suggestion
  ......rrrrr......
  ....rr.....rr....
  ...r.........r...
  ..r...........r..
  .r.............r.
  r...............r
```
With the structure rotated (manually, `orient.up` is only in placement) it gives this:
![image](https://github.com/user-attachments/assets/01673597-3dca-48ee-a472-389d8d9d4d33)